db/version_util.h

```diff
@@ -28,7 +28,9 @@ class OfflineManifestWriter {
                   /*db_id*/ "", /*db_session_id*/ "") {}
 
   Status Recover(const std::vector<ColumnFamilyDescriptor>& column_families) {
-    return versions_.Recover(column_families);
+    return versions_.Recover(column_families, /*read_only*/ false,
+                             /*db_id*/ nullptr,
+                             /*no_error_if_files_missing*/ true);
```

Comment:
should it be exposed an option for the command?

Reply:
When I read the comment on `OfflineManifestWriter`, I interpret it as wanting to operate on the manifest in a raw way (do not assume the referenced files exist).

And I don't think `no_error_if_files_missing` makes sense to users of `update_manifest`, because it checks for existence of the files regardless, nor `unsafe_remove_sst_file`, because it is intended to bypass safeguards anyway.